tox.ini

```diff
@@ -64,13 +64,32 @@ commands =
         -m pytest --ignore={[vars]tst_path}integration -v --tb native -s {posargs}
     coverage report
 
-[testenv:integration]
-description = Run integration tests
+[testenv:charm-integration]
+description = Run charm integration tests
 deps =
     pytest
     juju
     pytest-operator
     -r{toxinidir}/requirements.txt
 commands =
-    pytest -v --tb native --ignore={[vars]tst_path}unit --log-cli-level=INFO -s {posargs} --durations=0 --asyncio-mode=auto
+    pytest -v --tb native  {[vars]tst_path}integration/test_charm.py --log-cli-level=INFO -s {posargs} --durations=0 --asyncio-mode=auto
```

Comment:
Optional - could be worth using `--ignore` in the future instead of pointing to a test file, in case new the integration test scope grows and new files get added in addition to the `test_charm.py`

Reply:
True 🤔